src/protocols/apple-m0110/keyboard_interface.c

```
@@ -71,392 +71,418 @@
+    last_response_time = board_millis();  // Record response time for timeout management
+    __dmb();                              // Memory barrier - ensure volatile write completes
+
+    switch (keyboard_state) {
+        case UNINITIALISED:
+            // Should not receive data in UNINITIALISED state - ignore
+            LOG_ERROR("M0110 received data in UNINITIALISED state: 0x%02X\n", data_byte);
+            break;
+        case INIT_MODEL_REQUEST:
+            // Handle model number response during initialization phase
+            {
+                const char* model_desc = get_model_description(data_byte);
```

Clang (14.0.6)

[warning] variable 'model_desc' is not initialized

(cppcoreguidelines-init-variables)